### PR TITLE
#30 plural translation fixes

### PR DESCRIPTION
**Pull request by DarjaLeonova**

### CODE REVIEW

File: pkg/convert/pot.go [hunk 1]
```diff
@@ -53,6 +53,10 @@ func FromPot(b []byte) (model.Messages, error) {
 	messages := make([]model.Message, 0, len(po.Messages))
 
 	for _, node := range po.Messages {
+		if len(node.MsgStr) == 1 && node.MsgStr[0] == "\n" {
```
[the remark this answers is not in the document]
Comment (DarjaLeonova) [reply pkg/convert/pot.go:56]: In case if we have ID and no messages

[the remark this answers is not in the document]
Comment (janishorsts) [reply pkg/convert/pot.go:56]: example, please! :)

[the remark this answers is not in the document]
Comment (DarjaLeonova) [reply pkg/convert/pot.go]: I was wrong about "\n", I just forgot what syntax rules were in PO file format.
Right now all should be fixed without hardcoding the first msgstr with "\n"

File: pkg/convert/pot_test.go [hunk 2]
```diff
@@ -560,6 +563,98 @@ func TestFromPot(t *testing.T) {
...
+						Description: "a farewell",
+						Fuzzy:       false,
+					},
+				},
+			},
+		},
+		{
+			name: "msgid and msgstr empty headers",
+			input: []byte(`#, fuzzy
+							"Language: en\n"
```
Comment (janishorsts) [inline pkg/convert/pot_test.go:601]: This does not look right. Should it be?

```suggestion
							Language: en
```

Comment (DarjaLeonova) [reply pkg/convert/pot_test.go:601]: In all examples, that I found Language header ends with "\n", so I thought it should be mandatory.

https://www.drupal.org/community/contributor-guide/reference-information/localize-drupal-org/working-with-offline/po-and-pot-files
https://gist.github.com/bosskovic/5930785
https://www.simultrans.com/blog/what-are-.po-files

**Review by janishorsts — APPROVED** [review]

**Pull request merged**